yeonjae/programmers-1/삼진법뒤집기.js

```diff
@@ -0,0 +1,5 @@
+function solution(n) {
+  let result = n.toString(3).split("").reverse().join("");
+  
+  return parseInt(result, 3)
+}
```

Comment:
```suggestion
function solution(n) {
  const result = n.toString(3).split("").reverse().join("");
  
  return parseInt(result, 3)
}
```
- 잘푸셨네요!! 그래도 재할당이 필요없는 경우에는 가장 먼저 const로 선언해주세요 !! 

```js
function solution(n) {
  return parseInt(n.toString(3).split("").reverse().join(""), 3);
}
```
- 애초에 result를 선언할 필요없이 한 줄로 반환도 가능합니다 참고만해주세요

Reply:
@ssi02014 확인했습니다! 자꾸 습관처럼 let을 쓰네요 제가,,신경쓰겠습니당, 감사해요 :)